itests/util/src/main/java/org/apache/hadoop/hive/cli/control/CliConfigs.java

```diff
@@ -756,4 +756,40 @@ public KuduNegativeCliConfig() {
       }
     }
   }
+
+  public static class IcebergCliConfig extends AbstractCliConfig {
+
+    public IcebergCliConfig() {
+      super(CoreCliDriver.class);
+      try {
+        setQueryDir("iceberg/iceberg-handler/src/test/java/queries/positive");
```

Comment:
Shouldn't the `.q` files go under `src/test/queries/...` instead of `src/test/java/queries/...`?

Reply:
You are right, this is a typo

Reply:
Right, but the q files need to be moved too I think